util/gconv/gconv_time.go

```diff
@@ -72,8 +72,13 @@ func GTime(any interface{}, format ...string) *gtime.Time {
 	}
 	// Priority conversion using given format.
 	if len(format) > 0 {
-		t, _ := gtime.StrToTimeFormat(s, format[0])
-		return t
+		for _, item := range format {
+			t, err := gtime.StrToTimeFormat(s, item)
+			if t != nil && err == nil {
+				return t
+			}
+		}
+		return nil
```

Comment:
There is another function `ParseTimeFromContent` should also be updated, and associated the comments of the functions should also be updated.